lib/docker-delta.coffee

```diff
@@ -10,7 +10,7 @@ btrfs = require './btrfs'
 utils = require './utils'
 Docker = require 'docker-toolbelt'
 
-docker = new Docker()
+docker = new Docker(socketPath: process.env.DOCKER_SOCKET ? '/var/run/docker.sock')
```

Comment:
Well, that depends - dockerode examples use DOCKER_SOCKET https://github.com/apocas/dockerode/search?utf8=%E2%9C%93&q=DOCKER_SOCKET
And we've been using this variable in resin-io/resin-supervisor forever, I'd imagine there was an original reason for that?